engine/installation/linux/docker-ee/centos.md

```diff
@@ -0,0 +1,289 @@
+                  docker-selinux \
+                  docker-engine \
+                  docker-ce
+```
+
+It's OK if `yum` reports that none of these packages are installed.
+
+The contents of `/var/lib/docker/`, including images, containers, volumes, and
+networks, are preserved. The Docker EE package is now called `docker-ee`.
+
+## Install Docker EE
+
+You can install Docker EE in different ways, depending on your needs:
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the RPM package and install it manually and manage
+  upgrades completely manually. This is useful in situations such as installing
+  Docker on air-gapped systems with no access to the internet.
+
+### Install using the repository
+
+Before you install Docker EE for the first time on a new host machine, you need
+to set up the Docker EE repository. Afterward, you can install and update Docker
+EE from the repository.
+
+#### Set up the repository
+
+1.  Remove any existing Docker repositories from `/etc/yum.repos.d/`.
+
+2.  Store your Docker EE repository URL in a `yum` variable in `/etc/yum/vars/`.
+    Replace `<DOCKER-EE-URL>` with the URL you noted down in the
+    [prerequisites](#prerequisites).
+
+    ```bash
+    $ sudo sh -c 'echo "<DOCKER-EE-URL>" > /etc/yum/vars/dockerurl'
+    ```
+
+3.  Install required packages. `yum-utils` provides the `yum-config-manager`
+    utility, and `device-mapper-persistent-data` and `lvm2` are required by the
+    `devicemapper` storage driver.
+
+    ```bash
+    $ sudo yum install -y yum-utils device-mapper-persistent-data lvm2
+    ```
+
+4.  Use the following command to add the **stable** repository:
+
+    ```bash
+    $ sudo yum-config-manager \
+        --add-repo \
+        <DOCKER-EE-URL>/docker-ee.repo
+    ```
+
+#### Install Docker EE
+
+1.  Update the `yum` package index.
+
+    ```bash
+    $ sudo yum makecache fast
+    ```
+
+    If this is the first time you have refreshed the package index since adding
+    the Docker repositories, you will be prompted to accept the GPG key, and
+    the key's fingerprint will be shown. Verify that the fingerprint is
+    correct, and if so, accept the key. The fingerprint should match
+    `DD91 1E99 5A64 A202 E859  07D6 BC14 F10B 6D08 5F96`.
+
+2.  Install the latest version of Docker EE, or go to the next step to install a
+    specific version.
+
+    ```bash
+    $ sudo yum install docker-ee
+    ```
+
+    > **Warning**:
+    > If you have multiple Docker repositories enabled, installing
+    > or updating without specifying a version in the `yum install` or
+    > `yum update` command will always install the highest possible version,
+    > which may not be appropriate for your stability needs.
+    {:.warning}
+
+3.  On production systems, you should install a specific version of Docker EE
```

Comment:
Probably for a follow up; given that for Docker EE we have a separate directory in the repo for a specific version (e.g. 17.03 https://download.docker.com/linux/centos/7/x86_64/stable/Packages/) I think we should recommend to use that as URL for the repository, then people can upgrade and always be sure to get the latest patch release for the version they have installed.

Contrary to upgrading between releases (17.03 -> 17.06), upgrading to the latest _patch_ release may be a good thing to do, especially given that patch releases are only released for _major_ issues, or security fixes